Blink stops after </>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Blink stops doing its thing after </> and not until \</f>

### Theme

```json
        {
          "foreground": "#DCB977",
          "style": "plain",
          "template": " <f><#CB4B16>[x</>{{ .Meaning }}<#CB4B16>]</></f>",
          "type": "exit"
        }
```
![09_04_2023_wezterm-gui_zQBmpyEGRU](https://user-images.githubusercontent.com/34986564/230749393-957f9473-28f8-4816-b6e5-4a2ae9445267.gif)

### What OS are you seeing the problem on?

_No response_

### Which shell are you using?

_No response_

### Log output

```Shell
Version: 14.24.0

Shell: bash (5.1.16(1)-release)

Segments:

ConsoleTitle(true)                         -   0 ms -  Manager :: 
session(true)                              -   0 ms - ΓöÅ[hostname user]
sysinfo(true)                              -  27 ms - [RAM: 10/17GB]
git(true)                                  -  58 ms - [εéámaster ~2]
root(false)                                -   0 ms - 
node(false)                                -   3 ms - 
rust(false)                                -   1 ms - 
python(true)                               -   2 ms - εê╡  base 
spotify(true)                              -   5 ms - εé░ εÿé song 
executiontime(false)                       -   0 ms - 
project(false)                             -   0 ms - 
exit(false)                                -   0 ms - 
root(false)                                -   0 ms - 
text(true)                                 -   0 ms - Γ¥»

Run duration: 144.0136ms

Cache path: C:\Users\user\AppData\Local\oh-my-posh

Config path: C:\Users\user\.dotter/theme.omp.json

Logs:

[DEBUG] 03:14:49.533 shell.go:Getenv:392 ΓåÆ WezTerm
[TRACE] 03:14:49.533 shell.go:Getenv(TERM_PROGRAM) - 0s
[DEBUG] 03:14:49.533 shell.go:Getenv:392 ΓåÆ C:\Users\user\AppData\Local
[TRACE] 03:14:49.533 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 03:14:49.533 shell_windows.go:CachePath() - 0s
[DEBUG] 03:14:49.570 shell.go:Getenv:392 ΓåÆ C:\Users\user\.dotter/theme.omp.json
[TRACE] 03:14:49.570 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 03:14:49.574 shell.go:Shell:644 ΓåÆ process name: bash.exe
[TRACE] 03:14:49.574 shell.go:Shell() - 4.3829ms
[DEBUG] 03:14:49.575 shell.go:resolveConfigPath:338 ΓåÆ Cygwin detected, using full path for config
[TRACE] 03:14:49.575 shell.go:resolveConfigPath() - 5.3851ms
[TRACE] 03:14:49.575 shell.go:Init() - 42.3119ms
[TRACE] 03:14:49.575 shell.go:Flags() - 0s
[TRACE] 03:14:49.575 config.go:loadConfig() - 0s
[TRACE] 03:14:49.575 shell.go:Flags() - 0s
[DEBUG] 03:14:49.575 shell.go:Getenv:392 ΓåÆ NO DATA
[TRACE] 03:14:49.575 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 03:14:49.575 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[TRACE] 03:14:49.575 shell.go:GOOS() - 0s
[DEBUG] 03:14:49.575 shell_windows.go:WindowsRegistryKeyValue:206 ΓåÆ ColorizationColor(DWORD): 0xC4515C6B
[TRACE] 03:14:49.575 shell.go:Shell() - 0s
[DEBUG] 03:14:49.575 shell.go:Getenv:392 ΓåÆ 5.1.16(1)-release
[TRACE] 03:14:49.576 shell.go:Getenv(POSH_SHELL_VERSION) - 0s
[TRACE] 03:14:49.576 shell_windows.go:Root() - 0s
[TRACE] 03:14:49.576 shell.go:Shell() - 0s
[TRACE] 03:14:49.576 shell.go:ErrorCode() - 0s
[TRACE] 03:14:49.576 shell_windows.go:IsWsl() - 0s
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[DEBUG] 03:14:49.576 shell.go:Pwd:422 ΓåÆ R:\git\Manager
[TRACE] 03:14:49.576 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.576 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[DEBUG] 03:14:49.576 shell.go:User:548 ΓåÆ user
[TRACE] 03:14:49.576 shell.go:User() - 0s
[DEBUG] 03:14:49.576 shell.go:Host:560 ΓåÆ NDR-PC
[TRACE] 03:14:49.576 shell.go:Host() - 0s
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[TRACE] 03:14:49.576 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.576 shell.go:Flags() - 0s
[TRACE] 03:14:49.576 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.576 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.576 shell.go:GOOS() - 0s
[DEBUG] 03:14:49.576 shell.go:Getenv:392 ΓåÆ NO DATA
[TRACE] 03:14:49.576 shell.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 03:14:49.576 shell.go:Getenv:392 ΓåÆ NO DATA
[TRACE] 03:14:49.576 shell.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 03:14:49.576 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.576 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.576 shell.go:Shell() - 0s
[TRACE] 03:14:49.604 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.604 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.604 shell.go:GOOS() - 0s
[TRACE] 03:14:49.604 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.604 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.604 shell.go:Shell() - 0s
[TRACE] 03:14:49.604 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.604 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.604 shell.go:GOOS() - 0s
[TRACE] 03:14:49.604 shell.go:GOOS() - 0s
[DEBUG] 03:14:49.616 shell.go:CommandPath:599 ΓåÆ C:\program files\git\cmd\git.exe
[TRACE] 03:14:49.616 shell.go:CommandPath(git.exe) - 11.972ms
[TRACE] 03:14:49.616 shell.go:HasCommand(git.exe) - 11.972ms
[TRACE] 03:14:49.616 shell.go:Pwd() - 0s
[TRACE] 03:14:49.616 shell.go:HasParentFilePath(.git) - 0s
[DEBUG] 03:14:49.616 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.616 shell.go:GOOS() - 0s
[TRACE] 03:14:49.616 shell.go:GOOS() - 0s
[TRACE] 03:14:49.616 shell.go:GOOS() - 0s
[TRACE] 03:14:49.616 shell.go:GOOS() - 0s
[TRACE] 03:14:49.616 shell.go:GOOS() - 0s
[TRACE] 03:14:49.616 shell.go:GOOS() - 0s
[TRACE] 03:14:49.616 shell.go:GOOS() - 0s
[TRACE] 03:14:49.616 shell.go:GOOS() - 0s
[TRACE] 03:14:49.616 shell.go:GOOS() - 0s
[DEBUG] 03:14:49.616 shell.go:Getenv:392 ΓåÆ NO DATA
[TRACE] 03:14:49.616 shell.go:Getenv(POSH_GIT_STATUS) - 0s
[DEBUG] 03:14:49.616 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.616 shell.go:GOOS() - 0s
[DEBUG] 03:14:49.661 shell.go:RunCommand:578 Γåô
    # branch.oid 62e9d81853b2636905b4d82767c52630effe41c1
    # branch.head master
    # branch.upstream origin/master
    # branch.ab +0 -0
    1 .M N... 100644 100644 100644 a5011528fa44e64b4b96f04e97e324c29cea5f52 a5011528fa44e64b4b96f04e97e324c29cea5f52 Manager/pyinstaller.spec
    1 .M N... 100644 100644 100644 59b9863eae56fb8ba300bb8c19edde070e4a4c48 59b9863eae56fb8ba300bb8c19edde070e4a4c48 Manager/requirements.txt
[TRACE] 03:14:49.661 shell.go:RunCommand(git.exe -C R:/Remote --no-optional-locks -c core.quotepath=false -c color.status=false status -unormal --branch --porcelain=2) - 44.6815ms
[DEBUG] 03:14:49.661 shell.go:HasFolder:487 ΓåÆ false
[TRACE] 03:14:49.661 shell.go:HasFolder(R:\git\.git/rebase-merge) - 0s
[DEBUG] 03:14:49.661 shell.go:HasFolder:487 ΓåÆ false
[TRACE] 03:14:49.661 shell.go:HasFolder(R:\git\.git/rebase-apply) - 0s
[DEBUG] 03:14:49.661 shell.go:HasFilesInDir:458 ΓåÆ false
[TRACE] 03:14:49.661 shell.go:HasFilesInDir(MERGE_MSG) - 0s
[DEBUG] 03:14:49.661 shell.go:HasFilesInDir:458 ΓåÆ false
[TRACE] 03:14:49.661 shell.go:HasFilesInDir(CHERRY_PICK_HEAD) - 0s
[DEBUG] 03:14:49.661 shell.go:HasFilesInDir:458 ΓåÆ false
[TRACE] 03:14:49.661 shell.go:HasFilesInDir(REVERT_HEAD) - 0s
[DEBUG] 03:14:49.661 shell.go:HasFilesInDir:458 ΓåÆ false
[TRACE] 03:14:49.661 shell.go:HasFilesInDir(sequencer/todo) - 0s
[TRACE] 03:14:49.662 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.662 shell.go:TemplateCache() - 0s
[ERROR] 03:14:49.663 shell.go:FileContent:512 ΓåÆ open R:\git\.git/logs/refs/stash: The system cannot find the file specified.
[TRACE] 03:14:49.663 shell.go:FileContent(R:\git\.git/logs/refs/stash) - 1ms
[TRACE] 03:14:49.663 shell.go:Shell() - 0s
[TRACE] 03:14:49.663 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.663 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.663 shell.go:GOOS() - 0s
[TRACE] 03:14:49.663 shell_windows.go:Root() - 0s
[TRACE] 03:14:49.663 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.663 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.663 shell.go:GOOS() - 0s
[TRACE] 03:14:49.663 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.663 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.663 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.663 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.663 shell.go:HasFiles(*.js) - 0s
[TRACE] 03:14:49.663 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.663 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.663 shell.go:HasFiles(*.ts) - 0s
[TRACE] 03:14:49.663 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.663 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.663 shell.go:HasFiles(package.json) - 0s
[TRACE] 03:14:49.663 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.665 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.665 shell.go:HasFiles(.nvmrc) - 1.9999ms
[TRACE] 03:14:49.665 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.665 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.665 shell.go:HasFiles(pnpm-workspace.yaml) - 0s
[TRACE] 03:14:49.665 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.665 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.665 shell.go:HasFiles(.pnpmfile.cjs) - 0s
[TRACE] 03:14:49.665 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.666 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.666 shell.go:HasFiles(.npmrc) - 1.0001ms
[TRACE] 03:14:49.666 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.666 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.666 shell.go:HasFiles(.vue) - 0s
[TRACE] 03:14:49.666 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.666 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.667 shell.go:GOOS() - 0s
[TRACE] 03:14:49.667 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.667 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.667 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.667 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.667 shell.go:HasFiles(*.rs) - 0s
[TRACE] 03:14:49.667 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.667 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.667 shell.go:HasFiles(Cargo.toml) - 0s
[TRACE] 03:14:49.667 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.667 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.667 shell.go:HasFiles(Cargo.lock) - 0s
[TRACE] 03:14:49.667 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.667 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.667 shell.go:GOOS() - 0s
[TRACE] 03:14:49.667 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.667 shell_windows.go:56 ΓåÆ C:\Users\user
[DEBUG] 03:14:49.667 shell.go:Getenv:392 ΓåÆ NO DATA
[TRACE] 03:14:49.667 shell.go:Getenv(VIRTUAL_ENV) - 0s
[DEBUG] 03:14:49.667 shell.go:Getenv:392 ΓåÆ NO DATA
[TRACE] 03:14:49.667 shell.go:Getenv(CONDA_ENV_PATH) - 0s
[DEBUG] 03:14:49.667 shell.go:Getenv:392 ΓåÆ base
[TRACE] 03:14:49.667 shell.go:Getenv(CONDA_DEFAULT_ENV) - 0s
[TRACE] 03:14:49.667 shell.go:GOOS() - 0s
[TRACE] 03:14:49.667 shell.go:GOOS() - 0s
[TRACE] 03:14:49.667 shell.go:GOOS() - 0s
[TRACE] 03:14:49.667 shell.go:GOOS() - 0s
[TRACE] 03:14:49.667 shell.go:GOOS() - 0s
[TRACE] 03:14:49.667 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.667 shell.go:HasFiles:441 ΓåÆ true
[TRACE] 03:14:49.667 shell.go:HasFiles(*.py) - 0s
[TRACE] 03:14:49.667 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.669 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.670 shell.go:Shell() - 0s
[TRACE] 03:14:49.670 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.670 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.670 shell.go:GOOS() - 0s
[TRACE] 03:14:49.675 shell_windows.go:QueryWindowTitles(^(Spotify.*)|(.*\s-\s.*)$) - 5.0611ms
[TRACE] 03:14:49.675 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.675 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.675 shell.go:Shell() - 0s
[TRACE] 03:14:49.675 shell.go:Flags() - 0s
[TRACE] 03:14:49.675 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.675 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.675 shell.go:GOOS() - 0s
[TRACE] 03:14:49.675 shell.go:Pwd() - 0s
[TRACE] 03:14:49.675 shell.go:Shell() - 0s
[TRACE] 03:14:49.675 shell.go:Shell() - 0s
[DEBUG] 03:14:49.675 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.675 shell.go:GOOS() - 0s
[TRACE] 03:14:49.675 shell.go:GOOS() - 0s
[TRACE] 03:14:49.675 shell.go:PathSeparator() - 0s
[TRACE] 03:14:49.675 shell.go:GOOS() - 0s
[TRACE] 03:14:49.675 shell.go:GOOS() - 0s
[TRACE] 03:14:49.675 shell.go:GOOS() - 0s
[TRACE] 03:14:49.675 shell.go:GOOS() - 0s
[TRACE] 03:14:49.675 shell.go:GOOS() - 0s
[TRACE] 03:14:49.675 shell.go:GOOS() - 0s
[TRACE] 03:14:49.675 shell.go:GOOS() - 0s
[TRACE] 03:14:49.676 shell.go:PathSeparator() - 0s
[TRACE] 03:14:49.676 shell.go:PathSeparator() - 0s
[TRACE] 03:14:49.676 shell.go:GOOS() - 0s
[TRACE] 03:14:49.676 shell.go:GOOS() - 0s
[TRACE] 03:14:49.676 shell.go:GOOS() - 0s
[TRACE] 03:14:49.676 shell.go:PathSeparator() - 0s
[TRACE] 03:14:49.676 shell.go:GOOS() - 0s
[TRACE] 03:14:49.676 shell.go:GOOS() - 0s
[TRACE] 03:14:49.676 shell.go:GOOS() - 0s
[TRACE] 03:14:49.676 shell.go:PathSeparator() - 0s
[TRACE] 03:14:49.676 shell.go:GOOS() - 0s
[TRACE] 03:14:49.676 shell.go:GOOS() - 0s
[TRACE] 03:14:49.676 shell.go:GOOS() - 0s
[TRACE] 03:14:49.676 shell.go:PathSeparator() - 0s
[TRACE] 03:14:49.676 shell.go:PathSeparator() - 0s
[TRACE] 03:14:49.676 shell.go:PathSeparator() - 0s
[TRACE] 03:14:49.676 shell.go:PathSeparator() - 0s
[TRACE] 03:14:49.676 shell.go:Pwd() - 0s
[TRACE] 03:14:49.676 shell_windows.go:IsWsl() - 0s
[TRACE] 03:14:49.676 shell.go:StackCount() - 0s
[DEBUG] 03:14:49.676 win32_windows.go:isWriteable:303 ΓåÆ current user is member of S-1-5-32-544
[DEBUG] 03:14:49.676 win32_windows.go:isWriteable:311 Γåô
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 03:14:49.676 win32_windows.go:isWriteable:313 ΓåÆ user has write access
[TRACE] 03:14:49.676 shell_windows.go:DirIsWritable() - 391.5┬╡s
[TRACE] 03:14:49.676 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.677 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.677 shell.go:Shell() - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.677 shell.go:GOOS() - 0s
[TRACE] 03:14:49.677 shell.go:ExecutionTime() - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.677 shell.go:GOOS() - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.677 shell.go:HasFiles(package.json) - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.677 shell.go:HasFiles(Cargo.toml) - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.677 shell.go:HasFiles(pyproject.toml) - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.677 shell.go:HasFiles(composer.json) - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.677 shell.go:HasFiles(*.nuspec) - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.677 shell.go:HasFiles(*.vbproj) - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.677 shell.go:HasFiles(*.fsproj) - 549.2┬╡s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.677 shell.go:HasFiles(*.csproj) - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.677 shell.go:HasFiles(JuliaProject.toml) - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.677 shell.go:HasFiles(Project.toml) - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell.go:HasFiles:444 ΓåÆ false
[TRACE] 03:14:49.677 shell.go:HasFiles(*.psd1) - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.677 shell.go:GOOS() - 0s
[TRACE] 03:14:49.677 shell.go:ErrorCode() - 0s
[TRACE] 03:14:49.677 shell.go:ErrorCode() - 0s
[TRACE] 03:14:49.677 shell.go:Flags() - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.677 shell.go:GOOS() - 0s
[TRACE] 03:14:49.677 shell_windows.go:Root() - 0s
[TRACE] 03:14:49.677 shell.go:Pwd() - 0s
[DEBUG] 03:14:49.677 shell_windows.go:56 ΓåÆ C:\Users\user
[TRACE] 03:14:49.677 shell.go:GOOS() - 0s
[TRACE] 03:14:49.677 shell.go:TemplateCache() - 0s
[TRACE] 03:14:49.677 shell.go:Shell() - 0s
[DEBUG] 03:14:49.677 shell.go:Getenv:392 ΓåÆ C:\Users\user\AppData\Local
[TRACE] 03:14:49.677 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 03:14:49.677 shell_windows.go:CachePath() - 0s
[TRACE] 03:14:49.677 shell.go:Flags() - 0s
```

--- a/src/ansi/ansi_writer.go
+++ b/src/ansi/ansi_writer.go
@@ -24,7 +24,8 @@ var (
 		{AnchorStart: `<f>`, AnchorEnd: `</f>`, Start: "\x1b[5m", End: "\x1b[25m"},
 		{AnchorStart: `<r>`, AnchorEnd: `</r>`, Start: "\x1b[7m", End: "\x1b[27m"},
 	}
-	colorStyle = &style{AnchorStart: "COLOR", AnchorEnd: `</>`, End: "\x1b[0m"}
+	resetStyle      = &style{AnchorStart: "RESET", AnchorEnd: `</>`, End: "\x1b[0m"}
+	backgroundStyle = &style{AnchorStart: "BACKGROUND", AnchorEnd: `</>`, End: "\x1b[49m"}
 )
 
 type style struct {
@@ -335,7 +336,7 @@ func (w *Writer) Write(background, foreground, text string) {
 	w.hyperlinkState = OTHER
 
 	// reset colors
-	w.writeEscapedAnsiString(colorStyle.End)
+	w.writeEscapedAnsiString(resetStyle.End)
 
 	// reset current
 	w.currentBackground = ""
@@ -410,9 +411,9 @@ func (w *Writer) writeSegmentColors() {
 func (w *Writer) writeColorOverrides(match map[string]string, background string, i int) (position int) {
 	position = i
 	// check color reset first
-	if match[ANCHOR] == colorStyle.AnchorEnd {
+	if match[ANCHOR] == resetStyle.AnchorEnd {
 		// make sure to reset the colors if needed
-		position += len([]rune(colorStyle.AnchorEnd)) - 1
+		position += len([]rune(resetStyle.AnchorEnd)) - 1
 
 		// do not reset when colors are identical
 		if w.currentBackground == w.background && w.currentForeground == w.foreground {
@@ -429,7 +430,7 @@ func (w *Writer) writeColorOverrides(match map[string]string, background string,
 		}
 
 		if w.background.IsClear() {
-			w.writeEscapedAnsiString(colorStyle.End)
+			w.writeEscapedAnsiString(backgroundStyle.End)
 		}
 
 		if w.currentBackground != w.background && !w.background.IsClear() {

--- a/src/ansi/ansi_writer_test.go
+++ b/src/ansi/ansi_writer_test.go
@@ -32,7 +32,7 @@ func TestWriteANSIColors(t *testing.T) {
 		{
 			Case:     "Bold with color override",
 			Input:    "<b><#ffffff>test</></b>",
-			Expected: "\x1b[1m\x1b[30m\x1b[38;2;255;255;255mtest\x1b[0m\x1b[30m\x1b[22m\x1b[0m",
+			Expected: "\x1b[1m\x1b[30m\x1b[38;2;255;255;255mtest\x1b[49m\x1b[30m\x1b[22m\x1b[0m",
 			Colors:   &Colors{Foreground: "black", Background: ParentBackground},
 		},
 		{

